content_item_requests_test.go

```diff
@@ -0,0 +1,85 @@
+package main_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	. "github.com/alphagov/publishing-api"
+	. "github.com/alphagov/publishing-api/testhelpers"
```

Comment:
Using dot imports is generally not a recommended practice (it merges the namespaces, and can lead to interesting issues as a result).

IMO it's reasonable to use it for the test framework itself (ie ginkgo and gomega) as these define a DSL to use in the tests, and are designed to be used in this manner (although some purists would even object to that).

Regarding these imports, these tests are only using a single function from the `main` package (`BuildHTTPMux`), so that could easily not be a dot import.

I'm more on the fence over dot importing the test helpers.  In other places (eg the router test suite), I've build test helpers in the same package as the test suite (although in separate files), which is effectively the same as they're all in the same namespace.